### PR TITLE
Fix `containerProps` in `dcc.Markdown`

### PR DESCRIPTION
**Pull request by chriddyp**

### CODE REVIEW

**Comment by chriddyp** [comment]
Discovered while QA'ing the docs in https://github.com/plotly/dash-docs/pull/477

**Comment by chriddyp** [comment]
Fullscreen doesnt return correctly.
ddk-full-screen

**Comment by chriddyp** [comment]
DDK modal doesn't return correctly.
ddk-modal

**Comment by chriddyp** [comment]
This hover feels slower.
slower-hover

File: src/components/Markdown.react.js [hunk 1]
```diff
@@ -20,11 +21,12 @@ function DashMarkdown(props) {
                 (props.loading_state && props.loading_state.is_loading) ||
                 undefined
             }
+            {...propOr({}, 'containerProps', props)}
         >
             <Markdown
                 source={props.children}
                 escapeHtml={!props.dangerously_allow_html}
-                {...props}
+                {...omit(['containerProps', props)}
```
Comment (byronz) [inline src/components/Markdown.react.js]: @chriddyp  typo here caused the build failure

**Pull request merged**